tabs/src/components/app/views/runner/QueryInput.jsx

```diff
@@ -1,7 +1,8 @@
 
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { requestTypes, graphVersions, GRAPH_URL } from '../../../TabConstants';
-import { Button, Input, Flex, Dropdown } from '@fluentui/react-northstar';
+import { Button, Flex, Dropdown } from '@fluentui/react-northstar';
+import { TextField } from 'office-ui-fabric-react';
```

Comment:
I wonder if there's a way to use something from fluent-ui (probably not), as the textfield does look a bit out of place

Reply:
Actually, there is, but when I talked with Charles, he said that they aren't using fluentui yet, and he had me change it.
For parity, I think we shouldn't use it either, then

Reply:
oooOOh maybe ask @anneguo3 , because then i'm not sure, as they stressed using the teams app react lib alot to us LOL

Reply:
If this works - let's just merge it and make note of it either in the code or in documentation somewhere?